compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -208,7 +208,7 @@ object messages {
 
   case class WildcardOnTypeArgumentNotAllowedOnNew()(implicit ctx: Context)
   extends Message(WildcardOnTypeArgumentNotAllowedOnNewID) {
-    val kind = "syntax"
+    val kind = "Syntax"
```

Comment:
This was the only place where this string started with a lowercase. I assumed it was a typo, if somebody could confirm it, it would be good.

Reply:
Good